common/Cargo.toml

```diff
@@ -30,3 +30,4 @@ tari_test_utils = { version = "^0.0", path = "../infrastructure/test_utils"}
 serde = { version = "1.0.106", features = ["derive"] }
 anyhow = "1.0"
 toml = "0.5"
+tari_wallet = { path = "../base_layer/wallet", version = "^0.1" }
```

Comment:
I might be misunderstanding but is this necessary? I don't see a reason for it below and I would be hesitant to introduce this dependency if its not needed?

Reply:
Thanks yes, seems I missed removing that.